### ISSUE TITLE
Netlify access audit, cleanup, and next steps.

### ISSUE THREAD

**Issue by divya-mohan0209** [kubernetes/community#7311]
### Describe the issue

Per [this issue](https://github.com/kubernetes/website/issues/38839) and wider discussions in the Release retrospective + SIG Docs meetings, we have realised that an audit & cleanup of privileged accesses to Netlify is long overdue. 

We've initiated a cleanup of Docs emeritus leadership and the efforts are being tracked [here](https://github.com/kubernetes/website/issues/41199).

Since SIG Contribex handles the granting and revoking of access for other areas of the project, this issue has been created for the following purposes in the order mentioned below.

- [ ] Seeking consensus around such an initiative
- [ ] Establishing a timeframe for periodic review of accesses
- [ ] Documenting accesses & access levels in a central document under k/community
- [ ] Automating the process of granting access via a GitHub PR (nice-to-have)

cc: @reylejano @natalisucks

**Comment by divya-mohan0209** [comment]
@cblecker : I don't know if tagging you here would be appropriate. Apologies, if not. This is about the Netlify Access Amendment PR you had commented on earlier and specifically, this issue relates to non-docs accounts.

**Comment by divya-mohan0209** [comment]
@Priyankasaggu11929 @palnabarun @cblecker : Tagging y'all on this since it has been pending for quite some time. What are your views on the above, if any?

**Comment by divya-mohan0209** [comment]
@Priyankasaggu11929 Thanks for the follow-up. All of these items are pending and would require further discussion. Happy to present this on a future SIG ContribEx call, if need be.

**Comment by divya-mohan0209** [comment]
@Priyankasaggu11929 I'll try making it to the APAC-friendly one tomorrow.

**Comment by reylejano** [comment]
Through Slack messages, we will create a netlify-admins group
During the creation of this group, we can address some of the items in this issue:

>  Establishing a timeframe for periodic review of accesses
Documenting accesses & access levels in a central document under k/community
Automating the process of granting access via a GitHub PR (nice-to-have)

Link to Slack threads:
- https://kubernetes.slack.com/archives/CCK68P2Q2/p1751309489340729
- https://kubernetes.slack.com/archives/C1J0BPD2M/p1750933450889639

**Comment by BenTheElder** [comment]
> Since SIG Contribex handles the granting and revoking of access for other areas of the project, this issue has been created for the following purposes in the order mentioned below.

That is true for community platforms like slack, but for most of our infrastructure we have been doing that through https://github.com/kubernetes/k8s.io under SIG K8s Infra, automated where possible.

I'd argue that Netlify predated SIG K8s Infra, but today managing access, setting up automation and billing observability, etc to cloud services / infrastructure that are then delegated to other SIGs is clearly in the K8s Infra charter. I don't think netlify is a contributor platform analogous to slack/discuss/github, it's a content hosting service which only site admins need access to.

Or put another way: If tomorrow netlify stopped funding us, where would we obtain alternate hosting?
That would almost certainly be through SIG K8s Infra to obtain and bootstrap the necessary resources and then delegate infrastructure access to the relevant groups.

I would at least like to make sure infrastructure leads have access, this is currently a visibility gap, we don't know the scale of this service and what it might take to replace it, unlike the project's other vendors.

Spinning up a new site also involves other changes (at least DNS) which are in k8s.io, unfortunately it hasn't been clear where we can direct people to obtain a netlify site currently, whereas we've been able to field all other infrastructure requests.

x-ref: https://github.com/kubernetes/k8s.io/issues/8124

cc @kubernetes/sig-k8s-infra-leads

**Comment by reylejano** [comment]
@BenTheElder , I'll bring this up in a SIG K8s Infra meeting

**Comment by palnabarun** [comment]
@reylejano was there any conclusion from the SIG K8s Infra meeting (if the discussion happened)? If not, what are the next steps?
